vars/CentOS_8.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+# by ansible used packages
```

Comment:
```suggestion
# packages used by ansible
```

Reply:
Maybe we should also fix the comment `RedHat.yml`.

The same configuration would probably also be relevant for RHEL 8 and we should create a file for that too.